src/debug_info.js

```diff
@@ -379,48 +397,62 @@ class DebugInfo {
                 elements: elements
             };
         }
-        
+
         return statement;
     }
 
     static isValidSymbol(token) {
-        return token.charAt(0) == '.' || token.endsWith(':');
+        if (token.charAt(0) == '.' || token.charAt(0) == '@' || token.charAt(0) == '_') {
+            return true;
+        }
+        var firstChar = token.charAt(0).toLowerCase();
+        if (firstChar < 'a' || firstChar > 'z') {
+            return false;
+        }
+        // instructions cannot be symbols or labels
+        if (token.length == 3
+            &&  ("adc,and,asl,bcc,bcs,beq,bit,bmi,bne,bpl,brk,bvc,bvs,clc,cld,cli,clv,cmp,cpx,cpy,dec,dex,"+
```

Comment:
Minor, but this might be better put to a global variable.

Reply:
OK